### PR TITLE
Replace Oasis headers with public domain headers

### LINKED ISSUE #76: LicenseRef-OASIS-IPR is considered non-free
The OASIS PKCS#11 header files that you have included are non-free.
Please consider replacing them with the drop-in replacements from the NSS library or pkcs11.h from OpenSC.

### PR DESCRIPTION
**Pull request by simo5**
Fixes #76

### CODE REVIEW

**Comment by simo5** [comment]
@jakuje @sahanaprasad07 can one of you review?
Ignore the Debian and MacOS failures, they are due to the fact both these distributions upgraded to the broken NSS 3.94 we fixed already in Fedora.

File: src/pkcs11.h [hunk 1]
```diff
@@ -1,29 +1,2411 @@
...
- *   with 1-byte alignment. In a UNIX environment, it may or may not be necessary
- *   (or even possible) to alter the byte-alignment of structures.
- *
- * Thus, use default alignment for Crypto structures for Linux. This is also
- * the defacto standard among all other users and implementations of PKCS#11.
- */
-#include "oasis/pkcs11.h"
-
-#endif /* P11PROV_PKCS11_H_ */
+/* This file is in the Public Domain */
```
Comment (bluca) [inline src/pkcs11.h:1]: Shouldn't this still use a SPDX line, pointing to the custom local identifier defined for this?

Comment (bluca) [reply src/pkcs11.h:1]: ie: `/* SPDX-License-Identifier: LicenseRef-Public-Domain-PKCS11-Headers */`

Comment (simo5) [reply src/pkcs11.h:1]: This is the untouched original file as downloaded form the pkcs11-headers project.

The connection between SPDX identifier and the file in in .reuse/dep5

**Review by Jakuje — APPROVED** [review]
lgtm

**Comment by simo5** [comment]
TY

**Pull request merged**